module/VuFind/src/VuFind/Recommend/Databases.php

```diff
@@ -105,6 +105,15 @@ class Databases implements RecommendInterface, \Psr\Log\LoggerAwareInterface
      */
     protected $useQueryMinLength = 3;
 
+    /**
+     * When using the query string as a match point, the query string and
+     * database names will first be normalized by removing the characters
+     * in this regular expression.
```

Comment:
Maybe add this comment?

```suggestion
     * in this regular expression. If empty, no normalization will occur.
```